cmd/osbuild-worker/main.go

```diff
@@ -237,6 +239,9 @@ func main() {
 		"koji-finalize": &KojiFinalizeJobImpl{
 			KojiServers: kojiServers,
 		},
+		"depsolve": &DepsolveJobImpl{
+			RPMMD: rpmmd.NewRPMMD(rpmmd_cache, "/usr/libexec/osbuild-composer/dnf-json"),
```

Comment:
The worker needs to pull in `dnf-json`. Put it in a sub-package that both composer and the worker depends on?